sdk/eventhub/azure-eventhub/azure/eventhub/aio/_eventprocessor/partition_context.py

```diff
@@ -74,7 +74,7 @@ async def update_checkpoint(self, event: Optional["EventData"] = None) -> None:
                     "offset": checkpoint_event.offset,
                     "sequence_number": checkpoint_event.sequence_number,
                 }
-                await self._checkpoint_store.update_checkpoint(checkpoint)
+                await self._checkpoint_store.update_checkpoint(checkpoint, **kwargs)
```

Comment:
the sync change needs to be applied here as well.
(my bad for forgetting about the async part in my previous review)